.changeset/config.json

```
@@ -6,5 +6,6 @@
   "linked": [],
   "ignore": [],
   "access": "public",
-  "baseBranch": "main"
+  "baseBranch": "main",
+  "onlyUpdatePeerDependentsWhenOutOfRange": true
```

Tror den må ligge i `__experimentalUnsafeOptions_WILL_CHANGE_IN_PATCH` (😅) for å funke
```suggestion
  "___experimentalUnsafeOptions_WILL_CHANGE_IN_PATCH": {
    "onlyUpdatePeerDependentsWhenOutOfRange": true
  }
```

---

duh! 😄